src/.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use CodeIgniter\CodingStandard\CodeIgniter4;
+use Nexus\CsConfig\Factory;
+use PhpCsFixer\Finder;
+
+$finder = Finder::create()
+    ->files()
+    ->in(__DIR__)
```

Comment:
Isn't `__DIR__` too broad? And if someone runs without modification, it seems a bit danger.

```suggestion
    ->in([
        // __DIR__ . '/app',
        // __DIR__ . '/src',
        __DIR__ . '/tests',
    ])
```

Reply:
Yes good suggestion. I'm defaulting Config files to use **app/** since I expect more developers are making projects, but down the road I'd like a script to convert them for libraries using **src/**.